src/containerapp/HISTORY.rst

```diff
@@ -5,6 +5,7 @@ Release History
 0.3.31
 ++++++
 * Fix issue when using 'az containerapp up' to create a container app from a local source with a Dockerfile
+* Fix issue for 'az containerapp job update' when updating a job with trigger configurations
```

Comment:
You cannot add new content to the already released extension version, Please add the description in the updated extension version.